app/backend/src/couchers/servicers/requests.py

```diff
@@ -436,36 +437,38 @@ def GetHostRequestUpdates(self, request, context):
             if request.newest_message_id == 0:
                 context.abort(grpc.StatusCode.INVALID_ARGUMENT, errors.INVALID_MESSAGE)
 
-            newest_message = session.query(Message).filter(Message.id == request.newest_message_id).one_or_none()
+            if not session.execute(select(Message).where(Message.id == request.newest_message_id)).scalar_one_or_none():
+                context.abort(grpc.StatusCode.INVALID_ARGUMENT, errors.INVALID_MESSAGE)
```

Comment:
I'm not sure what's going on here. But I think this RPC is being cut anyway.

Reply:
I interpreted it as .one_or_none() would raise an exception if the wrong result was found, so I added an explicit abort. Couldn't figure out what else this was doing here.